index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CodeJam1-CV</title>
+    <link rel="stylesheet" href="https://meyerweb.com/eric/tools/css/reset/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+    integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+</head>
+
+<body>
+
+    <div class="wrapper">
+      <header class="header">
+
+          <img class="photo" src="images/photo.png" alt="photo" >
+
+
+        <h1>Maksim Kalachou</h1>
+        <h2>Junior JS Dev</h2>
+        <p class="contacts">
+        Minsk/Mogilev, Belarus<br>
+        <i class="fas fa-phone-volume"></i>
+        <i class="fab fa-viber"></i>
+        <i class="fab fa-telegram">+375 (29) 742 78 89</i>
+        <br>
+        <i class="fab fa-skype"> zurzurm</i><br>
+        <i class="fab fa-github-square"> <a href="https://zurzur.github.com">zurzur.github.com</a></i>
+      </p>
+      </header>
+
+      <main>
+        <section>
+          <h3>SUMMARY</h3>
+          Since I recognized computers first time I was intresting in web and its tecnologies.
+          It took me consider for a while =) but now my main aim is to become Junior Web Developer.
+        </section>
+        <section>
+          <h3>SKILLS</h3>
+            <h4>Languages:</h4>
+            JavaScript, HTML/CSS, Java, Python, C++, SQL.
+            <h4>Bug Tracking Control:</h4>
+            Jira
+            <h4>Version Control:</h4>
+            GIT, SVN
+            <h4>RDBMS:</h4>
+            MySQL
+            <h4>Virtualisation Tools:</h4>
+            Virtual Box
+            <h4>OSs:</h4>
+            Linux, Windows, Android
+        </section>
+        <section>
+          <h3>EXAMPLES OF CODE</h3>
+            <a href="https://zurzur.github.com">On my GitHub account</a><br>
+            <a href="https://codewars.com/users/zurzur" target="_blank">On my CodeWars account</a>
+        </section>
+        <section>
+          <h3>EXPERIENCE IN PROGRAMMING</h3>
+            Match-match web game (JS, HTML/CSS);<br>
+            Hook for corporate SVN(Subversion) (Python);<br>
+            Several small Java and Android applications for private use (JAVA, MySQL);
+        </section>
+        <section>
+          <h3>EDUCATION</h3>
+          <ul>
+            <li>
+            Oriental Linguistic and Country Studies (Arabic Region);<br>
+            Belarusian State University, Faculty of International Relations<br>
+            Minsk, Belarus
+
```

Comment:
Unclosed `<li>` tag in several places of the file
It always should be closed
`<li></li>`